src/code/z_room.c

```diff
@@ -167,45 +164,45 @@ void func_80095D04(GlobalContext* globalCtx, Room* room, u32 flags) {
         }
     }
 
-    iREG(87) = polygon2->num;
+    iREG(87) = polygon2->num & 0xffff & 0xffff & 0xffff; // if this is real then I might not be
```

Comment:
```suggestion
    iREG(87) = polygon2->num & 0xFFFF & 0xFFFF & 0xFFFF; // if this is real then I might not be
```
at least you can be unreal in all caps